app/mailers/host_mailer.rb

```diff
@@ -47,6 +47,17 @@ def error_state(report)
     group_mail(owners, :subject => (_("Puppet error on %s") % host.to_label))
   end
 
+  def host_built(host)
+    host = Host.find(host)
+    owners = host.owner.recipients_for(:host_built)
```

Comment:
Do take 69943393 into account here, it's probably best to move it to a private method.